src/main.rs

```diff
@@ -6,6 +6,7 @@ use std::io::{Write, stderr, stdin};
 use std::process::{Command, exit};
 use std::path::PathBuf;
 use std::str::FromStr;
+use std::io::Read;
```

Comment:
Move this in with the other imports from `std::io` on row 5. Actually, with the other changes I suggested `Read`, isn't used anymore, but `BufRead` and `BufReader` is.

```suggestion
```

```diff
- use std::io::{Write, stderr, stdin};
+ use std::io::{BufRead, BufReader, Write, stderr, stdin};
```